sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -250,8 +250,8 @@ case class ConcatWs(children: Seq[Expression])
     Examples:
       > SELECT _FUNC_(1, 'scala', 'java');
        scala
-      > SELECT _FUNC_(1, encode('scala', 'utf-8'), encode('java', 'utf-8'));
-       scala
+      > SELECT _FUNC_(2, 'a', 1);
```

Comment:
note that ELT can accept any input type.

Reply:
I see. Thank you.

Reply:
does this example involve type coercion?